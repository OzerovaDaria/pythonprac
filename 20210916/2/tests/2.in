[6523461, 999, 0, -2, 88]
