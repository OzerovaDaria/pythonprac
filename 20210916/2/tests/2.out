[-2, 0, 88, 999, 6523461]
